src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/client/ProfessionalApiClient.java

```diff
@@ -66,42 +67,43 @@ public String getHealthPage() {
                 .asString();
     }
 
-    @SuppressWarnings("unchecked")
-    public Map<String, Object> createOrganisation(String organisationName,
-                                                  String[] pbas) {
-
+    private  OrganisationCreationRequest.OrganisationCreationRequestBuilder createOrganisationRequest() {
         List<PbaAccountCreationRequest> pbaAccounts = asList(aPbaPaymentAccount()
-                                                                     .pbaNumber(pbas[0])
-                                                                     .build(),
-                                                             aPbaPaymentAccount()
-                                                                     .pbaNumber(pbas[1])
-                                                                     .build());
-
-        OrganisationCreationRequest organisationCreationRequest = someMinimalOrganisationRequest()
-                        .name(organisationName)
-                        .sraId(randomAlphabetic(10) + "sra-id-number1")
-                        .sraRegulated(Boolean.FALSE)
-                        .companyUrl(randomAlphabetic(10) + "company-url")
-                        .companyNumber(randomAlphabetic(5) + "com")
-                        .pbaAccounts(pbaAccounts)
-                        .superUser(aUserCreationRequest()
-                                .firstName("some-fname")
-                                .lastName("some-lname")
-                                .email(randomAlphabetic(10) + "@somewhere.com")
-                                .build())
-                        .contactInformation(Arrays.asList(aContactInformationCreationRequest()
-                                .addressLine1("addressLine1")
-                                .addressLine2("addressLine2")
-                                .addressLine3("addressLine3")
-                                .country("some-country")
-                                .county("some-county")
-                                .townCity("some-town-city")
-                                .postCode("some-post-code")
-                                .dxAddress(Arrays.asList(dxAddressCreationRequest()
-                                        .dxNumber("DX 1234567890")
-                                        .dxExchange("dxExchange").build()))
-                                .build()))
-                        .build();
+                .pbaNumber(randomAlphabetic(10))
+                .build(),
+            aPbaPaymentAccount()
+                .pbaNumber(randomAlphabetic(10))
+                .build());
+
+        return someMinimalOrganisationRequest()
+            .name(randomAlphabetic(10))
+            .sraId(randomAlphabetic(10) + "sra-id-number1")
```

Comment:
Status in the minimal organisation request is missing?

Reply:
Added.